app/assets/javascripts/application.js

```diff
@@ -238,6 +238,19 @@ var Downloads = {
     Downloads.observeGUIOSFilter();
     Downloads.observePopState();
     Downloads.filterGUIS();
+    Downloads.autoDownload();
+  },
+
+  autoDownload: function() {
+    var ref = document.referrer;
+    var src = $('#auto-download-link').attr('href');
+
+    if (ref && src && ref.indexOf(document.location.origin) === 0) {
+      $('.download .hide').removeClass('hide');
```

Comment:
I think this can be achieved with `$('.download .hide').show()`

Reply:
Changed per your instruction.